### ISSUE TITLE
Comments are associated with the wrong nodes

### ISSUE THREAD

**Issue by gabro** [buildo/metarpheus#46]
## description

currently metarpheus is too eager in finding related comments
## how to reproduce

``` scala
@enum trait {
  /** a comment about Foo */
  object Foo
  object Bar
}
```

metarpheus will associate `a comment about Foo` to both `Foo` and `Bar`
## specs

Idea: maybe check that there are no newlines between the comment and the definition

**Comment by gabro** [comment]
This causes https://github.com/buildo/metarpheus/issues/43

**Comment by gabro** [comment]
We lack scalameta support atm (https://gitter.im/scalameta/scalameta?at=57617482b8ad3d5d7ee06105)

**Comment by gabro** [comment]
scalameta added support! This will be fixed by #51
